src/intpack.f90

```diff
@@ -1367,16 +1367,13 @@ subroutine fmc(mvar,xvar,expmx,fmch)
    a=a-0.5d0
    approx=.886226925428d0*(dsqrt(xd)*xd**m)
    !!!if (m) 21, 23, 21
-   if(m.eq.0d0) then
-      goto 23
-   else
-      goto 21
+   if(m.ne.0d0) then
+      do i=1,m
+         b=b-1.d0
+         approx=approx*b
+      enddo
    endif
-   21  do i=1,m
-      b=b-1.d0
-      approx=approx*b
-   enddo
-   23  fimult=.5d0*expmx*xd
+   fimult=.5d0*expmx*xd
```

Comment:
`goto 23` was used to jump to the else branch of the condition `m != 0`.

Reply:
Could you please expand on what do you mean? For me, changes look correct.

Reply:
The change is correct, I was just taking notes while reviewing.